feature/calendar/src/main/kotlin/team/noweekend/feature/calendar/component/todoList/CalendarTodoBoard.kt

```diff
@@ -0,0 +1,107 @@
+package team.noweekend.feature.calendar.component.todoList
+
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.AnnotatedString
+import androidx.compose.ui.text.SpanStyle
+import androidx.compose.ui.text.buildAnnotatedString
+import androidx.compose.ui.text.withStyle
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import kotlinx.collections.immutable.ImmutableList
+import kotlinx.collections.immutable.persistentListOf
+import team.noweekend.core.common.ui.todo.model.Todo
+import team.noweekend.core.common.ui.todo.model.TodoType
+import team.noweekend.core.design.system.foundation.theme.NWKTheme
+
+@Composable
+fun TodoCalendarBoard(
+    todoList: ImmutableList<Todo>,
+    modifier: Modifier = Modifier,
+) {
+    Row(
+        modifier = modifier
+            .fillMaxWidth()
+            .height(54.dp).padding(top = 20.dp),
+    ) {
+        Text(
+            text = finishedBoard(todoList),
+            style = NWKTheme.typography.heading5,
+        )
+    }
+}
+
+
+@Composable
+private fun finishedBoard(
+    todoList: ImmutableList<Todo>,
+): AnnotatedString {
+    val isDonCount = todoList.count { it.isDone }
+    val totalCount = todoList.size
+    return buildAnnotatedString {
+        withStyle(
+            style = SpanStyle(
+                color = NWKTheme.color.Semantic.Text.neutral,
+            ),
+        ) {
+            append("할 일 $isDonCount")
+        }
+        withStyle(
+            style = SpanStyle(
+                color = NWKTheme.color.Neutral.neutralGray700,
+            ),
+        ) {
+            append(" / ")
+        }
+        withStyle(
+            style = SpanStyle(
+                color = NWKTheme.color.Semantic.Text.disabled,
+            ),
+        ) {
+            append("$totalCount ")
+        }
+        withStyle(
+            style = SpanStyle(
+                color = NWKTheme.color.Semantic.Text.neutral,
+            ),
+        ) {
+            append("개")
+        }
+    }
```

Comment:
p3 : 내가알기로 AnnotatedString을 만들 때 Text 컴포저블에 테마를 지정하면 기본 테마로 다 들어가고 필요한 부분에만 withStyle로 테마를 지정할 수 있을거야
확인한번 해보면 좋을듯?
```suggestion
    return buildAnnotatedString {
        append("할 일 $isDonCount")
        withStyle(
            style = SpanStyle(
                color = NWKTheme.color.Neutral.neutralGray700,
            ),
        ) {
            append(" / ")
        }
        withStyle(
            style = SpanStyle(
                color = NWKTheme.color.Semantic.Text.disabled,
            ),
        ) {
            append("$totalCount ")
        }
        append("개")
    }
```

```kotlin
Text(
    text = finishedBoard(todoList),
    style = NWKTheme.typography.heading5.copy(
        color = NWKTheme.color.Sementic.Text.neutral,
    ),
)
```

Reply:
@JaesungLeee 
[44e03a9](https://github.com/mash-up-kr/NoWeekend-Android/pull/81/commits/44e03a96fa11addf523bfb63d1f4a09f5e740e5a)